tools/installPythonClient.R

```diff
@@ -1,22 +1,19 @@
 # R script to install the Synapse Python client
 # The script itself is written in Python.  This is simply a wrapper to call it
 # via the rWithPython package
-# 
+#
 # Author: bhoff
 ###############################################################################
 
-library(PythonEmbedInR)
 args <- commandArgs(trailingOnly = TRUE)
 baseDir<-args[1]
 
 if (is.null(baseDir) || is.na(baseDir) || !file.exists(baseDir)) {
 	stop(paste("baseDir", baseDir, "is invalid"))
 }
-
-PythonEmbedInR::pyImport("sys")
-PythonEmbedInR::pyExec(sprintf("sys.path.append(\"%s\")", file.path(baseDir, "inst", "python")))
-
-PythonEmbedInR::pyImport("installPythonClient")
-
-command<-sprintf("installPythonClient.main('%s')", baseDir)
-PythonEmbedInR::pyExec(command)
+print("*** Using Python Configuration:")
+reticulate::py_config()
+reticulate::py_run_string("import sys")
+reticulate::py_run_string(sprintf("sys.path.append(\"%s\")", file.path(baseDir, "inst", "python")))
+reticulate::py_install(c("requests", "pandas", "pysftp", "jinja2", "markupsafe"))
+reticulate::py_install(c("synapseclient"), pip=T)
```

Comment:
Should we lock down the version of the client so that we are explicit about making upgrades?

Reply:
Yes.